runtime/src/system_instruction_processor.rs

```diff
@@ -72,6 +72,12 @@ fn assign_account_to_program(
         return Err(InstructionError::MissingRequiredSignature);
     }
 
+    // guard against sysvars being assigned
```

Comment:
this is already enforced by verify_instruction, I think...

Reply:
can you have create_account call this function so there's only one check?

Reply:
also: looks like the guard against changing anything about a sysvar account can be removed now